.circleci/config.yml

```diff
@@ -41,6 +44,8 @@ commands:
     steps:
       - run:
           name: Install go tools for testing
+          # update tools to latest release but clear the module cache after install
+          # to avoid conflicts with the main go dependencies.
```

Comment:
`main go dependencies` are those in `go.mod`?

is it possible to change `go.mod` to install `gotestsum`, `moq`, and so on?

i'm viewing this as a desirable change based on the preference to maintain the linter rc instead of params passed to the linter in the circle-ci yaml.  the linter rc is conceptually equivalent to `go.mod` in this comparison wherein we're attempting to shrink `.circleci/config.yaml` in favor of project-global (i.e. not just ci) setup.

Reply:
if managing dependencies is an issue, let's solve it separetely